<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SudokuMain</class>
 <widget class="QMainWindow" name="SudokuMain">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>455</width>
    <height>448</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>SudokuMain</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QSpinBox" name="tl11">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>40</y>
      <width>31</width>
      <height>27</height>
     </rect>
    </property>
    <property name="buttonSymbols">
     <enum>QAbstractSpinBox::NoButtons</enum>
    </property>
   </widget>
   <widget class="QSpinBox" name="tl31">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>100</y>
      <width>31</width>
      <height>27</height>
     </rect>
    </property>
    <property name="buttonSymbols">
     <enum>QAbstractSpinBox::NoButtons</enum>
    </property>
   </widget>
   <widget class="QSpinBox" name="lt21">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>70</y>
      <width>31</width>
      <height>27</height>
     </rect>
    </property>
    <property name="buttonSymbols">
     <enum>QAbstractSpinBox::NoButtons</enum>
    </property>
   </widget>
   <widget class="QSpinBox" name="tl12">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>40</y>
      <width>30</width>
      <height>27</height>
     </rect>
    </property>
    <property name="buttonSymbols">
     <enum>QAbstractSpinBox::NoButtons</enum>
    </property>
   </widget>
   <widget class="QSpinBox" name="lt22">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>70</y>
      <width>30</width>
      <height>27</height>
     </rect>
    </property>
    <property name="buttonSymbols">
     <enum>QAbstractSpinBox::NoButtons</enum>
    </property>
   </widget>
   <widget class="QSpinBox" name="tl32">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>100</y>
      <width>30</width>
      <height>27</height>
     </rect>
    </property>
    <property name="buttonSymbols">
     <enum>QAbstractSpinBox::NoButtons</enum>
    </property>
   </widget>
   <widget class="QSpinBox" name="tl13">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>40</y>
      <width>30</width>
      <height>27</height>
     </rect>
    </property>
    <property name="buttonSymbols">
     <enum>QAbstractSpinBox::NoButtons</enum>
    </property>
   </widget>
   <widget class="QSpinBox" name="tl23">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>70</y>
      <width>30</width>
      <height>27</height>
     </rect>
    </property>
    <property name="buttonSymbols">
     <enum>QAbstractSpinBox::NoButtons</enum>
    </property>
   </widget>
   <widget class="QSpinBox" name="tl33">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>100</y>
      <width>30</width>
      <height>27</height>
     </rect>
    </property>
    <property name="buttonSymbols">
     <enum>QAbstractSpinBox::NoButtons</enum>
    </property>
   </widget>
   <widget class="QSpinBox" name="tm22">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>70</y>
      <width>30</width>
      <height>27</height>
     </rect>
    </property>
    <property name="buttonSymbols">
     <enum>QAbstractSpinBox::NoButtons</enum>
    </property>
   </widget>
   <widget class="QSpinBox" name="tm33">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>100</y>
      <width>30</width>
      <height>27</height>
     </rect>
    </property>
    <property name="buttonSymbols">
     <enum>QAbstractSpinBox::NoButtons</enum>
    </property>
   </widget>
   <widget class="QSpinBox" name="tm11">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>40</y>
      <width>31</width>
      <height>27</height>
     </rect>
    </property>
    <property name="buttonSymbols">
     <enum>QAbstractSpinBox::NoButtons</enum>
    </property>
   </widget>
   <widget class="QSpinBox" name="tm13">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>40</y>
      <width>30</width>
      <height>27</height>
     </rect>
    </property>
    <property name="buttonSymbols">
     <enum>QAbstractSpinBox::NoButtons</enum>
    </property>
   </widget>
   <widget class="QSpinBox" name="tm12">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>40</y>
      <width>30</width>
      <height>27</height>
     </rect>
    </property>
    <property name="buttonSymbols">
     <enum>QAbstractSpinBox::NoButtons</enum>
    </property>
   </widget>
   <widget class="QSpinBox" name="tm32">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>100</y>
      <width>30</width>
      <height>27</height>
     </rect>
    </property>
    <property name="buttonSymbols">
     <enum>QAbstractSpinBox::NoButtons</enum>
    </property>
   </widget>
   <widget class="QSpinBox" name="tm23">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>70</y>
      <width>30</width>
      <height>27</height>
     </rect>
    </property>
    <property name="buttonSymbols">
     <enum>QAbstractSpinBox::NoButtons</enum>
    </property>
   </widget>
   <widget class="QSpinBox" name="tm21">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>70</y>
      <width>31</width>
      <height>27</height>
     </rect>
    </property>
    <property name="buttonSymbols">
     <enum>QAbstractSpinBox::NoButtons</enum>
    </property>
   </widget>
   <widget class="QSpinBox" name="tm31">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>100</y>
      <width>31</width>
      <height>27</height>
     </rect>
    </property>
    <property name="buttonSymbols">
     <enum>QAbstractSpinBox::NoButtons</enum>
    </property>
   </widget>
   <widget class="QSpinBox" name="tr22">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>70</y>
      <width>30</width>
      <height>27</height>
     </rect>
    </property>
    <property name="buttonSymbols">
     <enum>QAbstractSpinBox::NoButtons</enum>
    </property>
   </widget>
   <widget class="QSpinBox" name="tr33">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>100</y>
      <width>30</width>
      <height>27</height>
     </rect>
    </property>
    <property name="buttonSymbols">
     <enum>QAbstractSpinBox::NoButtons</enum>
    </property>
   </widget>
   <widget class="QSpinBox" name="tr1">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>40</y>
      <width>31</width>
      <height>27</height>
     </rect>
    </property>
    <property name="buttonSymbols">
     <enum>QAbstractSpinBox::NoButtons</enum>
    </property>
   </widget>
   <widget class="QSpinBox" name="tr13">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>40</y>
      <width>30</width>
      <height>27</height>
     </rect>
    </property>
    <property name="buttonSymbols">
     <enum>QAbstractSpinBox::NoButtons</enum>
    </property>
   </widget>
   <widget class="QSpinBox" name="tr12">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>40</y>
      <width>30</width>
      <height>27</height>
     </rect>
    </property>
    <property name="buttonSymbols">
     <enum>QAbstractSpinBox::NoButtons</enum>
    </property>
   </widget>
   <widget class="QSpinBox" name="tr32">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>100</y>
      <width>30</width>
      <height>27</height>
     </rect>
    </property>
    <property name="buttonSymbols">
     <enum>QAbstractSpinBox::NoButtons</enum>
    </property>
   </widget>
   <widget class="QSpinBox" name="tr23">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>70</y>
      <width>30</width>
      <height>27</height>
     </rect>
    </property>
    <property name="buttonSymbols">
     <enum>QAbstractSpinBox::NoButtons</enum>
    </property>
   </widget>
   <widget class="QSpinBox" name="tr21">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>70</y>
      <width>31</width>
      <height>27</height>
     </rect>
    </property>
    <property name="buttonSymbols">
     <enum>QAbstractSpinBox::NoButtons</enum>
    </property>
   </widget>
   <widget class="QSpinBox" name="tr31">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>100</y>
      <width>31</width>
      <height>27</height>
     </rect>
    </property>
    <property name="buttonSymbols">
     <enum>QAbstractSpinBox::NoButtons</enum>
    </property>
   </widget>
   <widget class="QSpinBox" name="ml22">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>170</y>
      <width>30</width>
      <height>27</height>
     </rect>
    </property>
    <property name="buttonSymbols">
     <enum>QAbstractSpinBox::NoButtons</enum>
    </property>
   </widget>
   <widget class="QSpinBox" name="ml33">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>200</y>
      <width>30</width>
      <height>27</height>
     </rect>
    </property>
    <property name="buttonSymbols">
     <enum>QAbstractSpinBox::NoButtons</enum>
    </property>
   </widget>
   <widget class="QSpinBox" name="ml11">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>140</y>
      <width>31</width>
      <height>27</height>
     </rect>
    </property>
    <property name="buttonSymbols">
     <enum>QAbstractSpinBox::NoButtons</enum>
    </property>
   </widget>
   <widget class="QSpinBox" name="ml13">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>140</y>
      <width>30</width>
      <height>27</height>
     </rect>
    </property>
    <property name="buttonSymbols">
     <enum>QAbstractSpinBox::NoButtons</enum>
    </property>
   </widget>
   <widget class="QSpinBox" name="ml12">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>140</y>
      <width>30</width>
      <height>27</height>
     </rect>
    </property>
    <property name="buttonSymbols">
     <enum>QAbstractSpinBox::NoButtons</enum>
    </property>
   </widget>
   <widget class="QSpinBox" name="ml32">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>200</y>
      <width>30</width>
      <height>27</height>
     </rect>
    </property>
    <property name="buttonSymbols">
     <enum>QAbstractSpinBox::NoButtons</enum>
    </property>
   </widget>
   <widget class="QSpinBox" name="ml23">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>170</y>
      <width>30</width>
      <height>27</height>
     </rect>
    </property>
    <property name="buttonSymbols">
     <enum>QAbstractSpinBox::NoButtons</enum>
    </property>
   </widget>
   <widget class="QSpinBox" name="ml21">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>170</y>
      <width>31</width>
      <height>27</height>
     </rect>
    </property>
    <property name="buttonSymbols">
     <enum>QAbstractSpinBox::NoButtons</enum>
    </property>
   </widget>
   <widget class="QSpinBox" name="ml31">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>200</y>
      <width>31</width>
      <height>27</height>
     </rect>
    </property>
    <property name="buttonSymbols">
     <enum>QAbstractSpinBox::NoButtons</enum>
    </property>
   </widget>
   <widget class="QSpinBox" name="mm22">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>170</y>
      <width>30</width>
      <height>27</height>
     </rect>
    </property>
    <property name="buttonSymbols">
     <enum>QAbstractSpinBox::NoButtons</enum>
    </property>
   </widget>
   <widget class="QSpinBox" name="mm33">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>200</y>
      <width>30</width>
      <height>27</height>
     </rect>
    </property>
    <property name="buttonSymbols">
     <enum>QAbstractSpinBox::NoButtons</enum>
    </property>
   </widget>
   <widget class="QSpinBox" name="mm11">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>140</y>
      <width>31</width>
      <height>27</height>
     </rect>
    </property>
    <property name="buttonSymbols">
     <enum>QAbstractSpinBox::NoButtons</enum>
    </property>
   </widget>
   <widget class="QSpinBox" name="mm13">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>140</y>
      <width>30</width>
      <height>27</height>
     </rect>
    </property>
    <property name="buttonSymbols">
     <enum>QAbstractSpinBox::NoButtons</enum>
    </property>
   </widget>
   <widget class="QSpinBox" name="mm12">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>140</y>
      <width>30</width>
      <height>27</height>
     </rect>
    </property>
    <property name="buttonSymbols">
     <enum>QAbstractSpinBox::NoButtons</enum>
    </property>
   </widget>
   <widget class="QSpinBox" name="mm32">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>200</y>
      <width>30</width>
      <height>27</height>
     </rect>
    </property>
    <property name="buttonSymbols">
     <enum>QAbstractSpinBox::NoButtons</enum>
    </property>
   </widget>
   <widget class="QSpinBox" name="mm23">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>170</y>
      <width>30</width>
      <height>27</height>
     </rect>
    </property>
    <property name="buttonSymbols">
     <enum>QAbstractSpinBox::NoButtons</enum>
    </property>
   </widget>
   <widget class="QSpinBox" name="mm21">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>170</y>
      <width>31</width>
      <height>27</height>
     </rect>
    </property>
    <property name="buttonSymbols">
     <enum>QAbstractSpinBox::NoButtons</enum>
    </property>
   </widget>
   <widget class="QSpinBox" name="mm31">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>200</y>
      <width>31</width>
      <height>27</height>
     </rect>
    </property>
    <property name="buttonSymbols">
     <enum>QAbstractSpinBox::NoButtons</enum>
    </property>
   </widget>
   <widget class="QSpinBox" name="mr22">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>170</y>
      <width>30</width>
      <height>27</height>
     </rect>
    </property>
    <property name="buttonSymbols">
     <enum>QAbstractSpinBox::NoButtons</enum>
    </property>
   </widget>
   <widget class="QSpinBox" name="mr33">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>200</y>
      <width>30</width>
      <height>27</height>
     </rect>
    </property>
    <property name="buttonSymbols">
     <enum>QAbstractSpinBox::NoButtons</enum>
    </property>
   </widget>
   <widget class="QSpinBox" name="mr11">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>140</y>
      <width>31</width>
      <height>27</height>
     </rect>
    </property>
    <property name="buttonSymbols">
     <enum>QAbstractSpinBox::NoButtons</enum>
    </property>
   </widget>
   <widget class="QSpinBox" name="mr13">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>140</y>
      <width>30</width>
      <height>27</height>
     </rect>
    </property>
    <property name="buttonSymbols">
     <enum>QAbstractSpinBox::NoButtons</enum>
    </property>
   </widget>
   <widget class="QSpinBox" name="mr12">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>140</y>
      <width>30</width>
      <height>27</height>
     </rect>
    </property>
    <property name="buttonSymbols">
     <enum>QAbstractSpinBox::NoButtons</enum>
    </property>
   </widget>
   <widget class="QSpinBox" name="mr32">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>200</y>
      <width>30</width>
      <height>27</height>
     </rect>
    </property>
    <property name="buttonSymbols">
     <enum>QAbstractSpinBox::NoButtons</enum>
    </property>
   </widget>
   <widget class="QSpinBox" name="mr23">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>170</y>
      <width>30</width>
      <height>27</height>
     </rect>
    </property>
    <property name="buttonSymbols">
     <enum>QAbstractSpinBox::NoButtons</enum>
    </property>
   </widget>
   <widget class="QSpinBox" name="mr21">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>170</y>
      <width>31</width>
      <height>27</height>
     </rect>
    </property>
    <property name="buttonSymbols">
     <enum>QAbstractSpinBox::NoButtons</enum>
    </property>
   </widget>
   <widget class="QSpinBox" name="mr31">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>200</y>
      <width>31</width>
      <height>27</height>
     </rect>
    </property>
    <property name="buttonSymbols">
     <enum>QAbstractSpinBox::NoButtons</enum>
    </property>
   </widget>
   <widget class="QSpinBox" name="dl22">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>270</y>
      <width>30</width>
      <height>27</height>
     </rect>
    </property>
    <property name="buttonSymbols">
     <enum>QAbstractSpinBox::NoButtons</enum>
    </property>
   </widget>
   <widget class="QSpinBox" name="dl33">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>300</y>
      <width>30</width>
      <height>27</height>
     </rect>
    </property>
    <property name="buttonSymbols">
     <enum>QAbstractSpinBox::NoButtons</enum>
    </property>
   </widget>
   <widget class="QSpinBox" name="dl11">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>240</y>
      <width>31</width>
      <height>27</height>
     </rect>
    </property>
    <property name="buttonSymbols">
     <enum>QAbstractSpinBox::NoButtons</enum>
    </property>
   </widget>
   <widget class="QSpinBox" name="dl13">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>240</y>
      <width>30</width>
      <height>27</height>
     </rect>
    </property>
    <property name="buttonSymbols">
     <enum>QAbstractSpinBox::NoButtons</enum>
    </property>
   </widget>
   <widget class="QSpinBox" name="dl12">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>240</y>
      <width>30</width>
      <height>27</height>
     </rect>
    </property>
    <property name="buttonSymbols">
     <enum>QAbstractSpinBox::NoButtons</enum>
    </property>
   </widget>
   <widget class="QSpinBox" name="dl32">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>300</y>
      <width>30</width>
      <height>27</height>
     </rect>
    </property>
    <property name="buttonSymbols">
     <enum>QAbstractSpinBox::NoButtons</enum>
    </property>
   </widget>
   <widget class="QSpinBox" name="dl23">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>270</y>
      <width>30</width>
      <height>27</height>
     </rect>
    </property>
    <property name="buttonSymbols">
     <enum>QAbstractSpinBox::NoButtons</enum>
    </property>
   </widget>
   <widget class="QSpinBox" name="dl21">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>270</y>
      <width>31</width>
      <height>27</height>
     </rect>
    </property>
    <property name="buttonSymbols">
     <enum>QAbstractSpinBox::NoButtons</enum>
    </property>
   </widget>
   <widget class="QSpinBox" name="dl31">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>300</y>
      <width>31</width>
      <height>27</height>
     </rect>
    </property>
    <property name="buttonSymbols">
     <enum>QAbstractSpinBox::NoButtons</enum>
    </property>
   </widget>
   <widget class="QSpinBox" name="dm22">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>270</y>
      <width>30</width>
      <height>27</height>
     </rect>
    </property>
    <property name="buttonSymbols">
     <enum>QAbstractSpinBox::NoButtons</enum>
    </property>
   </widget>
   <widget class="QSpinBox" name="dm33">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>300</y>
      <width>30</width>
      <height>27</height>
     </rect>
    </property>
    <property name="buttonSymbols">
     <enum>QAbstractSpinBox::NoButtons</enum>
    </property>
   </widget>
   <widget class="QSpinBox" name="dm11">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>240</y>
      <width>31</width>
      <height>27</height>
     </rect>
    </property>
    <property name="buttonSymbols">
     <enum>QAbstractSpinBox::NoButtons</enum>
    </property>
   </widget>
   <widget class="QSpinBox" name="dm13">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>240</y>
      <width>30</width>
      <height>27</height>
     </rect>
    </property>
    <property name="buttonSymbols">
     <enum>QAbstractSpinBox::NoButtons</enum>
    </property>
   </widget>
   <widget class="QSpinBox" name="dm12">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>240</y>
      <width>30</width>
      <height>27</height>
     </rect>
    </property>
    <property name="buttonSymbols">
     <enum>QAbstractSpinBox::NoButtons</enum>
    </property>
   </widget>
   <widget class="QSpinBox" name="dm32">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>300</y>
      <width>30</width>
      <height>27</height>
     </rect>
    </property>
    <property name="buttonSymbols">
     <enum>QAbstractSpinBox::NoButtons</enum>
    </property>
   </widget>
   <widget class="QSpinBox" name="dm23">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>270</y>
      <width>30</width>
      <height>27</height>
     </rect>
    </property>
    <property name="buttonSymbols">
     <enum>QAbstractSpinBox::NoButtons</enum>
    </property>
   </widget>
   <widget class="QSpinBox" name="dm21">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>270</y>
      <width>31</width>
      <height>27</height>
     </rect>
    </property>
    <property name="buttonSymbols">
     <enum>QAbstractSpinBox::NoButtons</enum>
    </property>
   </widget>
   <widget class="QSpinBox" name="dm31">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>300</y>
      <width>31</width>
      <height>27</height>
     </rect>
    </property>
    <property name="buttonSymbols">
     <enum>QAbstractSpinBox::NoButtons</enum>
    </property>
   </widget>
   <widget class="QSpinBox" name="dr22">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>270</y>
      <width>30</width>
      <height>27</height>
     </rect>
    </property>
    <property name="buttonSymbols">
     <enum>QAbstractSpinBox::NoButtons</enum>
    </property>
   </widget>
   <widget class="QSpinBox" name="dr33">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>300</y>
      <width>30</width>
      <height>27</height>
     </rect>
    </property>
    <property name="buttonSymbols">
     <enum>QAbstractSpinBox::NoButtons</enum>
    </property>
   </widget>
   <widget class="QSpinBox" name="dr11">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>240</y>
      <width>31</width>
      <height>27</height>
     </rect>
    </property>
    <property name="buttonSymbols">
     <enum>QAbstractSpinBox::NoButtons</enum>
    </property>
   </widget>
   <widget class="QSpinBox" name="dr13">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>240</y>
      <width>30</width>
      <height>27</height>
     </rect>
    </property>
    <property name="buttonSymbols">
     <enum>QAbstractSpinBox::NoButtons</enum>
    </property>
   </widget>
   <widget class="QSpinBox" name="dr12">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>240</y>
      <width>30</width>
      <height>27</height>
     </rect>
    </property>
    <property name="buttonSymbols">
     <enum>QAbstractSpinBox::NoButtons</enum>
    </property>
   </widget>
   <widget class="QSpinBox" name="dr32">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>300</y>
      <width>30</width>
      <height>27</height>
     </rect>
    </property>
    <property name="buttonSymbols">
     <enum>QAbstractSpinBox::NoButtons</enum>
    </property>
   </widget>
   <widget class="QSpinBox" name="dr23">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>270</y>
      <width>30</width>
      <height>27</height>
     </rect>
    </property>
    <property name="buttonSymbols">
     <enum>QAbstractSpinBox::NoButtons</enum>
    </property>
   </widget>
   <widget class="QSpinBox" name="dr21">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>270</y>
      <width>31</width>
      <height>27</height>
     </rect>
    </property>
    <property name="buttonSymbols">
     <enum>QAbstractSpinBox::NoButtons</enum>
    </property>
   </widget>
   <widget class="QSpinBox" name="dr31">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>300</y>
      <width>31</width>
      <height>27</height>
     </rect>
    </property>
    <property name="buttonSymbols">
     <enum>QAbstractSpinBox::NoButtons</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>360</y>
      <width>99</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Check</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>455</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuDatei">
    <property name="title">
     <string>Datei</string>
    </property>
    <addaction name="actionNeu"/>
    <addaction name="actionSchlie_en"/>
   </widget>
   <addaction name="menuDatei"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionNeu">
   <property name="text">
    <string>Neu</string>
   </property>
  </action>
  <action name="actionSchlie_en">
   <property name="text">
    <string>Schließen</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
